src/lib.rs

```diff
@@ -12,18 +12,28 @@
 //! // Initialize wgpu
 //! let event_loop = EventLoop::new();
 //! let window = winit::window::Window::new(&event_loop).unwrap();
-//! let instance = wgpu::Instance::new(wgpu::Backends::PRIMARY);
-//! let surface = unsafe { instance.create_surface(&window) };
+//! let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
+//!     backends: wgpu::Backends::PRIMARY,
+//!     dx12_shader_compiler: wgpu::Dx12Compiler::default(),
+//! });
+//! let surface = unsafe {
+//!     instance
+//!         .create_surface(&window)
+//!         .expect("Failed to create wgpu surface")
+//! };
```

Comment:
```suggestion
//! let instance = wgpu::Instance::new(wgpu::InstanceDescriptor::default());
//! let surface = unsafe { instance.create_surface(&window).unwrap() };
```

Reply:
`InstanceDescriptor::default()` returns `Backends::all()` instead of `Backends::Primary` (maybe it should be changed to PRIMARY instead of all on the wgpu side?). Does smaa-rs work on webgl?